modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -4110,7 +4157,7 @@ private void checkNearCacheStarted(IgniteCacheProxy<?, ?> cache) throws IgniteCh
     }
 
     /**
-     * <tt>ctx.gateway().readLock()</tt>
+     * The <tt>ctx.gateway().readLock()</tt> is used underneath.
```

Comment:
Can we use `@code` instead of `<tt>`? Here and in a couple of places.

Reply:
Fixed all mentions.